### PR TITLE
Adding Alex Sorokoletov to the authors

### PR DESCRIPTION
**Pull request by alexsorokoletov**

### CODE REVIEW

**Comment by alexsorokoletov** [comment]
I was reading the guidance for pull requests and though: hey, how can someone forget to add file to csproj. 
Here I am :)

Alex

**Comment by jamesmontemagno** [comment]
You need to open it in VS/XS and add the c# file in like any other project :)

File: src/Firehose.Web/Authors/AlexandrSorokoletov.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web
+{
+    public class AlexandrSorokoletov : IAmAMicrosoftMVP
+    {
+        public string FirstName => "Alexandr";
+        public string LastName => "Sorokoletov";
+        public string ShortBioOrTagLine => "I plan, design and develop mobile products to achieve measurable business results";
```
Comment (ghuntley) [inline src/Firehose.Web/Authors/AlexandrSorokoletov.cs]: PlanetXamarin isn't LinkedIn or your resume. Have a look at the tone/voice of some of the other author's taglines.  Keep it human, not business. i.e. "Tomasz Cielecki is a Xamarin MVP who loves long walks on the beach, yelling at the screen. More importantly writes code."

File: src/Firehose.Web/Authors/AlexandrSorokoletov.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        public string StateOrRegion => "Washington, D.C.";
+        public string EmailAddress => string.Empty;
+        public string TwitterHandle => "AlexSorokoletov";
+        public string GitHubHandle => "alexsorokoletov";
+        public GeoPosition Position => new GeoPosition(38.905147, -77.065189);
+        public Uri WebSite => new Uri("https://sorokoletov.com");
+        public IEnumerable<Uri> FeedUris { get { yield return new Uri("https://sorokoletov.com/atom.xml"); } }
+        public string GravatarHash => "b07fef8827dd03655303751e2fd5ca95";
+        DateTime IAmAMicrosoftMVP.FirstAwarded => new DateTime(2013, 7, 1);
```
Comment (ghuntley) [inline src/Firehose.Web/Authors/AlexandrSorokoletov.cs]: Drop this entire line, we don't need or use it anymore. My fault - need to update the instructions in README.md

File: src/Firehose.Web/Authors/AlexandrSorokoletov.cs (hunk 1 above)
Comment (ghuntley) [inline src/Firehose.Web/Authors/AlexandrSorokoletov.cs]: See http://sorokoletov.com/about/ for inspiration

>  enjoy snowboarding and kitesurfing. I like to travel, read books

**Comment by ghuntley** [comment]
(Oh also add the authors class to the csproj)

**Comment by alexsorokoletov** [comment]
Thanks, was unsure about what you're looking for in tagline. Checked existing authors but that didn't help to decide. 
Or maybe I wanted to have a chat with you, guys ;)

As soon as I get to computer, I'll fix it 

Alex

**Comment by alexsorokoletov** [comment]
Thank you, Geoffrey, for finishing touches!

**Pull request merged**